### PR TITLE
Add plugin version to server info

### PR DESCRIPTION
**Pull request by Aschen**
## What does this PR do ?

Load plugin version from `package.json` if it exists and returns it in `server:info`

### Other changes
 
  - use new framework for `server:getConfig` and `server:healthCheck` 

### Boyscout

 - async/await
 - remove obsolete default configuration (`queue` service)

### CODE REVIEW

File: lib/api/controllers/server.js [hunk 1]
```diff
@@ -89,67 +89,76 @@ class ServerController extends NativeController {
     // and default rights
     delete config.security;
 
-    return Bluebird.resolve(config);
+    if (config.VAULT_KEY) {
+      delete config.VAULT_KEY
+    }
+
+    return config
```
Comment (kuzzle) [inline lib/api/controllers/server.js]: MINOR Add a semicolon at the end of this statement. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)

File: lib/api/controllers/server.js [hunk 2]
```diff
@@ -89,67 +89,76 @@ class ServerController extends NativeController {
     // and default rights
     delete config.security;
 
-    return Bluebird.resolve(config);
+    if (config.VAULT_KEY) {
+      delete config.VAULT_KEY
```
Comment (kuzzle) [inline lib/api/controllers/server.js]: MINOR Add a semicolon at the end of this statement. [rule](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)

**Comment by kuzzle** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: test/core/plugins/manager/init.test.js [hunk 3]
```diff
@@ -32,7 +32,7 @@ describe('PluginsManager', () => {
     };
     Manifest = rewire('../../../../lib/core/plugins/manifest');
     Manifest.__set__({
-      fs: manifestFsStub
+      fs: manifestFsStub,
```
Comment (Aschen) [inline test/core/plugins/manager/init.test.js]:
```suggestion
      fs: manifestFsStub
```

File: test/core/plugins/manager/init.test.js [hunk 4]
```diff
@@ -46,7 +46,8 @@ describe('PluginsManager', () => {
     fsStub = {
       readdirSync: sinon.stub().returns([]),
       accessSync: sinon.stub(),
-      statSync: sinon.stub()
+      statSync: sinon.stub(),
+      existsSync: sinon.stub().returns(false),
```
Comment (Aschen) [inline test/core/plugins/manager/init.test.js]:
```suggestion
      existsSync: sinon.stub().returns(false)
```

**Pull request merged**